docs/creating-new/create/cmake.rst

```diff
@@ -275,7 +275,10 @@ Download release archive and calculate ``SHA1``:
 Add this information to ``cmake/projects/hunter_box_1/hunter.cmake`` file:
 
 .. code-block:: cmake
-  :emphasize-lines: 10, 12, 14, 16
+  :emphasize-lines: 1, 13, 15, 17, 19
+  
+  # Copyright (c) 2016-2020, Rahul Sheth, Ruslan Baratov
```

Comment:
This should be `2022` at the least. Perhaps it is better to insert `<your-name-here>` or some kind of placeholder so that contributors can put their names on the list.